OracleJava/17/Dockerfile

```diff
@@ -29,30 +28,31 @@ LABEL maintainer="Aurelio Garcia-Ribeyro <aurelio.garciaribeyro@oracle.com>"
 
 # Since the files are compressed as tar.gz first dnf install tar. gzip is already in oraclelinux:8
 RUN dnf install -y tar
-	
+
 # Default to UTF-8 file.encoding
 ENV LANG en_US.UTF-8
 
 # Environment variables for the builder image.
 # Required to validate that you are using the correct file
 
-ENV JAVA_URL=https://download.oracle.com/java/17/latest \
-	JAVA_HOME=/usr/java/jdk-17
+ENV JAVA_HOME=/usr/java/jdk-17
 
 ##
+COPY *.tar.gz /tmp/
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN set -eux; \
 	ARCH="$(uname -m)" && \
-	# Java uses just x64 in the name of the tarball
     if [ "$ARCH" = "x86_64" ]; \
-        then ARCH="x64"; \
+    then \
+		mv "$(ls /tmp/jdk-17*_linux-x64_bin.tar.gz)" /tmp/jdk.tar.gz ; \
```

Comment:
Very minor, but this `mv` is spaced over further than the else.

Reply:
Fixed in all files/lines